include/megaapi_impl.h

```diff
@@ -919,20 +919,20 @@ class MegaContactRequestPrivate : public MegaContactRequest
 
 #ifdef ENABLE_SYNC
 
-class MegaSyncEventPrivate: public MegaSyncEvent
+class MegaSyncEventPrivate : public MegaSyncEvent
 {
 public:
+    explicit
     MegaSyncEventPrivate(int type);
```

Comment:
this is just one declaration; it should be just one line.

Reply:
Is that part of our coding guideline?